cves/CVE-2011-1184.yml

```diff
@@ -116,10 +121,15 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
-  contest:
+  answer: |
+   The developers had found a separate weakness in the Tomcat's implementation for authentication 
+   about 3 months ago. They came out with a solution for that problem and when they 
```

Comment:
"about 3 months ago" -> "about 3 months prior", ago is relative and seems like you are talking about today.